tests/provision/user/main.fmf

```diff
@@ -0,0 +1,5 @@
+summary: Check that setting a user works
+adjust:
```

Comment:
Yeah, I will drop it, originally I wanted to also test `--user` with other methods, then realized it would just extend the scope of the change, which I did not want to do

Reply:
removed